frontend/src/metabase/query_builder/components/GuiQueryEditor.jsx

```diff
@@ -220,11 +252,11 @@ export default class GuiQueryEditor extends Component {
                     <BreakoutWidget
                         key={"breakout"+i}
                         className="View-section-breakout SelectionModule p1"
-                        fieldOptions={Query.getFieldOptions(tableMetadata.fields, true, tableMetadata.breakout_options.validFieldsFilter, _.omit(usedFields, breakout))}
```

Comment:
This is a great line to replaced. I think this is the sort of thing that makes the query-lib valuable.